<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingForm</class>
 <widget class="QWidget" name="SettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_StarRailSelect">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>30</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_StarRailPath">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_StarRailSetting">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_GenshinSelect">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>70</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_GenshinPath">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_GenshinSetting">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ZZZSelect">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>110</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ZZZPath">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ZZZSetting">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Honkai3Select">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>150</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Honkai3Path">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Honkai3Setting">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_ZZZPath_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_StarRailSelect_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>30</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Honkai3Select_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>150</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ZZZSelect_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>110</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_GenshinSelect_2">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>70</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Honkai3Setting_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_GenshinPath_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ZZZSetting_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Honkai3Path_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_StarRailPath_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>521</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_StarRailSetting_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>30</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_GenshinSetting_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>70</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Setting</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
